ClCd_XRotor.m

```diff
@@ -0,0 +1,76 @@
+% |Name        : ClCd_XRotor.m                                                           |
+% |Author      : Crescenzo Visone                                                        |
+% |              University of Naples Federico II.                                       |
+% |Version     : 1.02                                                                    |
+% |Date        : 25/11/2020                                                              |
+% |Modified    : 11/12/2020                                                              |
+% |Description : Cd(Cl) evaluation law according to XRotor aero model                    |     
+% |Reference   : XRotor User Guide                                                       |
+% |              <http://web.mit.edu/drela/Public/web/xrotor/xrotor_doc.txt>             |                                                         |
+% |Input       : * the first input must be a vector with the following components:       |
+% |              (Cd_min) = minimum blade element's drag coefficient                     |
+% |              (dCd_dCl2) = quadratic coefficient of the Cd(Cl) parable                |  
+% |              (Cl_Cd_min) = lift coefficient @ minimum Cd                             |
+% |              (Re_ref) = blade element's Reynolds number                              |
+% |              (Re_inf) = asymptotic Reynolds number                                   |
+% |              (f) = Reynolds number scaling exponent                                  |
+% |              (Cl_max) = maximum blade element's lift coefficient                     |
+% |              (Cl_min) = minimum blade element's lift coefficient                     |
+% |              * the second input can either be a scalar value or a vector equal to :  | 
+% |              (Cl_bp) = blade element's lift coefficient breakpoints                  |
+% |Output      : (Cl_vec) = lift coefficient vector used to plot the law                 |
+% |              (Cd) = drag coefficent values @ Cl_vec                                  |
+% |              (Cd_bp) = drag coefficient values @ requested Cl_bp                     |
+% |Note        :                                                                         |
+% ==============================================================================================
+
+    % taking input data
+    Cd_min    = input_v(1);
+    dCd_dCl2  = input_v(2);
+    Cl_Cd_min = input_v(3);
+    Re_ref    = input_v(4);
+    Re_inf    = input_v(5);
+    f         = input_v(6);
+    Cl_max    = input_v(7);
+    Cl_min    = input_v(8);
+    % lift coefficient vector creation
+    Cl_vec = Cl_min:.01:Cl_max;
+    % aerodynamic model building
+    for i = 1:length(Cl_vec)
+        Cl = Cl_vec(i);
+        Cd(i) = (Cd_min + dCd_dCl2*(Cl_Cd_min - Cl)^2)*(Re_inf/Re_ref)^f;
+    end
+    % plot
+    plot(Cd, Cl_vec);
```

Comment:
Put the plots at the end of the function and comment in a way that who has to do the output interface will understand.